api/build.gradle

```diff
@@ -30,7 +40,9 @@ dependencies {
 
     implementation libs.spring.security.ldap
 
-    implementation libs.kafka.clients
+    implementation (libs.kafka.clients) {
```

Comment:
This is tracked and fixed in https://issues.apache.org/jira/browse/KAFKA-19951, I assume that it may take a bit to land in Confluent's distribution. Can we document a TODO here to potentially remove this?  We may not even need the overwrite above after this is released depending on where else we use this library